miden-lib/asm/miden/kernels/tx/account.masm

```diff
@@ -60,12 +62,15 @@ const.MAX_SLOT_TYPE=64
 # Event emitted to signal that an account storage item is being updated.
 const.ACCOUNT_STORAGE_SET_ITEM_EVENT=131074
 
+# Event emitted to signal that an account storage map item is being updated.
+const.ACCOUNT_STORAGE_SET_MAP_ITEM_EVENT=131075
+
 # Event emitted to signal that account nonce is being incremented.
-const.ACCOUNT_INCREMENT_NONCE_EVENT=131075
+const.ACCOUNT_INCREMENT_NONCE_EVENT=131076
 
 # Event emitted to push the index of the account procedure at the top of the operand stack onto
 # the advice stack.
-const.ACCOUNT_PUSH_PROCEDURE_INDEX_EVENT=131076
+const.ACCOUNT_PUSH_PROCEDURE_INDEX_EVENT=131077
```

Comment:
Have you checked the other files? There may be an error in another file with the same index. I think the new error define above should instead use the next available idx instead of updating everything.

Reply:
I did check, and I moved the other even codes accordingly, see https://github.com/0xPolygonMiden/miden-base/blob/57185284e8db55d4f7f74781072309e131adad45/miden-lib/src/transaction/events.rs#L17